cli/hzn.go

```diff
@@ -541,8 +541,10 @@ Environment Variables:
 	mmsObjectDeleteType := mmsObjectDeleteCmd.Flag("type", msgPrinter.Sprintf("The type of the object to delete.")).Short('t').Required().String()
 	mmsObjectDeleteId := mmsObjectDeleteCmd.Flag("id", msgPrinter.Sprintf("The id of the object to delete.")).Short('i').Required().String()
 	mmsObjectDownloadCmd := mmsObjectCmd.Command("download", msgPrinter.Sprintf("Download data of the given object in the Horizon Model Management Service."))
-	mmsObjectDownloadType := mmsObjectDownloadCmd.Flag("type", msgPrinter.Sprintf("The type of the object to download data. This flag must be used with -i.")).Short('t').String()
-	mmsObjectDownloadId := mmsObjectDownloadCmd.Flag("id", msgPrinter.Sprintf("The id of the object to download data. This flag must be used with -t.")).Short('i').String()
+	mmsObjectDownloadType := mmsObjectDownloadCmd.Flag("type", msgPrinter.Sprintf("The type of the object to download data. This flag must be used with -i.")).Short('t').Required().String()
+	mmsObjectDownloadId := mmsObjectDownloadCmd.Flag("id", msgPrinter.Sprintf("The id of the object to download data. This flag must be used with -t.")).Short('i').Required().String()
+	mmsObjectDownloadFileName := mmsObjectDownloadCmd.Flag("fileName", msgPrinter.Sprintf("The file that the data of downloaded object is written to. This flag must be used with -f. If omit, will use default file name in format of {objectType}_{objectID}")).Short('f').String()
+	mmsObjectDownloadFilePath := mmsObjectDownloadCmd.Flag("path", msgPrinter.Sprintf("The path of the file that stores the data of downloaded object. This flag must be used with -p. If omit, will save file to current directory")).Short('p').String()
```

Comment:
Can you combin the -f  and -p into one flag -f?  So the user will specify something like: -f /usr/me/filename.
If /usr/me/ is a directory, then save the output to /usr/me/filename.  If /usr/me/filename is a directory, save the output to /usr/me/filename/objType_objId file.  If -f is omitted, save the output to current_dir/objType_objId.